modules/integration/tests-integration/tests-backend/src/test/resources/org/wso2/identity/integration/test/rest/api/server/idp/v1/empty-custom-fed-auth.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
AFAIR, we discussed, that we do not need this. Instead of doing, we discussed to add delete test case.

Reply:
Yes, the file is deleted with the commit 38608298188e4987329c170552b4a0de9fb72daf